schemas/clinvar-study.tables.json

```diff
@@ -0,0 +1,3068 @@
+            "name": "position_vcf",
+            "datatype": "integer"
+          },
+          {
+            "name": "reference_allele_vcf",
+            "datatype": "string"
+          },
+          {
+            "name": "alternate_allele_vcf",
+            "datatype": "string"
+          },
+          {
+            "name": "for_display_length",
+            "datatype": "boolean"
+          }
+        ]
+      },
+      {
+        "name": "type_chromosome",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "type_chromosome_nr",
+            "datatype": "string"
+          },
+          {
+            "name": "type_chromosome_str",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_chromosome_nr",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "chromosome",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_chromosome_str",
+        "columns": [
+          {
+            "name": "persistent_id",
+            "datatype": "string"
+          },
+          {
+            "name": "chromosome",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "haploin_sufficiency",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "haploin_sufficiency",
+            "datatype": "string"
+          },
+          {
+            "name": "last_evaluated",
+            "datatype": "date"
+          },
+          {
+            "name": "clingen",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "triplo_sensitivity",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "triplo_sensitivity",
+            "datatype": "string"
+          },
+          {
+            "name": "last_evaluated",
+            "datatype": "date"
+          },
+          {
+            "name": "clingen",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_xref",
+        "columns": [
+          {
+            "name": "uid",
+            "datatype": "string"
+          },
+          {
+            "name": "db",
+            "datatype": "string"
+          },
+          {
+            "name": "id",
+            "datatype": "string"
+          },
+          {
+            "name": "type",
+            "datatype": "string"
+          },
+          {
+            "name": "url",
+            "datatype": "string"
+          },
+          {
+            "name": "status",
+            "datatype": "string"
+          }
+        ]
+      },
+      {
+        "name": "type_status",
```

Comment:
This table could go

Reply:
Should go. This is just an enum. `type_xref.status `should just be the string value of the enum.